app/.env.example

```
@@ -1,2 +1,2 @@
-VITE_CONTRACT_ADDRESS=""
+VITE_CONTRACT_ADDRESS="0xA3E91f209D40949A23d5b54e8A4949E578fdeB0E"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)